tests/playwright/config/global-setup.js

```diff
@@ -7,8 +7,9 @@ module.exports = async ( config ) => {
 	await page.goto( `${ config.projects[ 0 ].use.baseURL }/wp-admin` );
 
 	await page.waitForSelector( 'text=Log In' );
-	await page.fill( 'input[name="log"]', 'admin' );
-	await page.fill( 'input[name="pwd"]', 'password' );
+	await page.fill( 'input[name="log"]', process.env.UNAME || 'admin' );
+	await page.waitForTimeout( 500 );
```

Comment:
Why is a 500ms delay required?

Reply:
Because sometimes playwright is confusing, and puts the password in the username input for some reason, unless it has a 500ms "time to rest".